extensions/schema-compare/package.json

```diff
@@ -13,7 +13,7 @@
   "icon": "images/sqlserver.png",
   "aiKey": "AIF-37eefaf0-8022-4671-a3fb-64752724682e",
   "activationEvents": [
-    "*"
+    "onCommand:schemaCompare.start"
```

Comment:
Looks good, only thing I am worried about is - experience wise will it take too long to open sc page because of additional loading?

Reply:
generally extensions can start within a few seconds. We can always tune perf in the future.